app/javascript/components/taggingWrapper.jsx

```diff
@@ -3,57 +3,59 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Spinner } from 'patternfly-react';
 import { TaggingWithButtonsConnected, TaggingConnected, taggingApp } from '@manageiq/react-ui-components/dist/tagging';
-import { http } from '../http_api';
 
 const params = (type = 'default', state, tag = {}) => ({
   provision: {
-    id: "new",
+    id: 'new',
     ids_checked: [state.tagging.appState.assignedTags.map(t => t.values.map(val => val.id)).flat(), tag.tagValue.id].flat(),
-    tree_typ: 'tags'
+    tree_typ: 'tags',
   },
   default: {
-    id: state.tagging.appState.affectedItems[0] || "new",
+    id: state.tagging.appState.affectedItems[0] || 'new',
     cat: tag.tagCategory.id,
     val: tag.tagValue.id,
     check: 1,
-    tree_typ: 'tags'
-  }
-})[type]
+    tree_typ: 'tags',
+  },
+})[type];
 
-const onDelete = (type = 'default', params = [], deleted_element) => ({
-  provision: () => ({...params, check: 0, ids_checked: params.ids_checked.filter(element => element !== deleted_element) }),
-  default: ()   => params,
-})[type]
+const onDelete = (type = 'default', params = [], deletedElement) => ({
+  provision: () => ({ ...params, check: 0, ids_checked: params.ids_checked.filter(element => element !== deletedElement) }),
+  default: () => params,
+})[type];
 
 class TaggingWrapper extends React.Component {
   constructor(props) {
     super(props);
-    ManageIQ.redux.addReducer({tagging: taggingApp});
+    ManageIQ.redux.addReducer({ tagging: taggingApp });
   }
 
   componentDidMount() {
     this.loadState(this.props.tags);
   }
 
   loadState = state => this.props.loadState(state);
+
   reset = () => this.props.reset();
+
   isLoaded = () => this.props.isLoaded();
 
   render() {
     if (!this.props.isLoaded) return <Spinner loading size="lg" />;
     const { urls, options, tagging } = this.props;
-    return (options && options.hideButtons && <TaggingConnected options={{...options, params, onDelete }}/> || <TaggingWithButtonsConnected
+    return (options && options.hideButtons && (<TaggingConnected options={{ ...options, params, onDelete }} /> || (
+    <TaggingWithButtonsConnected
       saveButton={{
-          // FIXME: jQuery is necessary here as it communicates with the old world
-          // don't replace $.post with http.post
-          onClick: (assignedTags) => {
-            $.post(urls.save_url, { data: JSON.stringify(assignedTags) });
-          },
-          href: '',
-          type: 'button',
-          disabled: _.isEqual({...tagging.initialState, selected: undefined}, {...tagging.appState, selected: undefined}),
-          description: __('Save'),
-        }
+        // FIXME: jQuery is necessary here as it communicates with the old world
+        // don't replace $.post with http.post
+        onClick: (assignedTags) => {
```

Comment:
I believe that the onClick callback should be passed as an outside prop. Or make the whole `saveButton` props override able with spreading the rest of the props to the component.

Reply:
I can do it, but it is not my code, I was just going through it to get the idea of how to use the redux store. I just reformatted, since it had 60+ linter errors (and still has a few but I didn't want to change the structure). Should I remove from here create another PR and refactor it?

Reply:
I think it's out of the context of this PR.